### PR TITLE
Update usage text for virtual-workspaces to be accurate

### PR DESCRIPTION
**Pull request by hasheddan**
## Summary

Updates the usage text for the virtual-workspaces command to be accurate
with current status.

Signed-off-by: hasheddan <georgedanielmangum@gmail.com>

## Related issue(s)

n/a -- good to see you folks :)

### CODE REVIEW

File: cmd/virtual-workspaces/command/cmd.go [hunk 1]
```diff
@@ -55,8 +55,8 @@ func NewCommand(ctx context.Context, errout io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "workspaces",
-		Short: "Launch workspaces virtual workspace apiserver",
-		Long:  "Start a virtual workspace apiserver to managing personal, shared or organization workspaces",
+		Short: "Launch virtual workspace apiserver",
+		Long:  "Start a virtual workspace apiserver to manage personal, shared or organization workspaces",
```
Comment (sttts) [inline cmd/virtual-workspaces/command/cmd.go:59]: actually this line is long outdated.

Comment (sttts) [inline cmd/virtual-workspaces/command/cmd.go:59]:
```suggestion
		Short: "Launch virtual workspace apiservers",
		Long:  "Start virtual workspace apiserver for different purposes.",
```

Comment (hasheddan) [reply cmd/virtual-workspaces/command/cmd.go:59]: That should future-proof it for a while :)

Comment (sttts) [reply cmd/virtual-workspaces/command/cmd.go:59]: not sure this is so much better. It runs these https://github.com/kcp-dev/kcp/tree/main/pkg/virtual: syncer, workspaces, apiexport, initializingworkspaces. More to come.

Comment (hasheddan) [reply cmd/virtual-workspaces/command/cmd.go:59]: @sttts what would you think about making the `Long` description say something about starting the root API server to be a bit more specific?

Comment (sttts) [reply cmd/virtual-workspaces/command/cmd.go:59]: Fine with me. Wondering whether we can get the list easily somehow.

Comment (sttts) [reply cmd/virtual-workspaces/command/cmd.go:59]: It's probably hidden deep in the constructor code.

Comment (hasheddan) [reply cmd/virtual-workspaces/command/cmd.go:59]: > Wondering whether we can get the list easily somehow.

Seems like a great thing to do in a codegen step 👍🏻 I'll update to just mention starting the root API server for now -- happy to open an issue tracking automating listing components in usage if that would be desirable

File: cmd/virtual-workspaces/command/cmd.go [hunk 2]
```diff
@@ -55,8 +55,8 @@ func NewCommand(ctx context.Context, errout io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "workspaces",
-		Short: "Launch workspaces virtual workspace apiserver",
-		Long:  "Start a virtual workspace apiserver to managing personal, shared or organization workspaces",
+		Short: "Launch virtual workspace apiservers",
+		Long:  "Start the root virtual workspace apiserver to enable virtual workspace management.",
```
Comment (hasheddan) [inline cmd/virtual-workspaces/command/cmd.go:59]: @sttts updated here -- happy to do more iteration if this feels under-specified or inaccurate

File: cmd/virtual-workspaces/command/cmd.go (hunk 1 above)
Comment (sttts) [reply cmd/virtual-workspaces/command/cmd.go]: :+1: to both

File: cmd/virtual-workspaces/command/cmd.go (hunk 2 above)
Comment (sttts) [reply cmd/virtual-workspaces/command/cmd.go:59]: It's not super important clearly. If you run this command manually, you know what you are doing.

**Review by davidfestal — APPROVED** [review]

**Pull request merged**